docs/docs/developers/guides/getting_started_on_testnet.md

```diff
@@ -37,7 +37,7 @@ bash -i <(curl -s https://install.aztec.network)
 Then install the version of the network running the testnet:
 
 ```bash
-aztec-up alpha-testnet
+aztec-up -v #include_testnet_version
```

Comment:
I had replaced these with `aztec-up latest`, assuming we want to keep this as a product feature where `latest` points to the version we want as latest. Thinking of the scenario where we wanted to rollback a release or few.

Reply:
(general point shared with other changes)